docs/book/src/plugins/available-plugins.md

```diff
@@ -2,20 +2,21 @@
+| Plugin                                                      | Key                  | Description                                                                                                                                                                                                                                    |
+| ----------------------------------------------------------- | -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [go.kubebuilder.io/v3](go-v3-plugin.md)                     | `go/v3`              | Default scaffold to create the projects when none plugin(s) are informed. Responsible for scaffold the Golang project and its configurations.                                                                                                  |
+| [go.kubebuilder.io/v2](go-v2-plugin.md)                     | `go/v2`              | Golang plugin responsible for scaffold the legacy layout provided with Kubebuilder CLI >= `2.0.0` and < `3.0.0`.                                                                                                                               |
+| [declarative.go.kubebuilder.io/v1](declarative-v1.md)       | `declarative/v1`     | Optional plugin used to scaffold APIs/controllers using the [kubebuilder-declarative-pattern][kubebuilder-declarative-pattern] project.                                                                                                        |
+| [kustomize.common.kubebuilder.io/v1](kustomize-v1.md)       | `kustomize/v1`       | Responsible for scaffold all manifests to configure the projects with [kustomize(v3)][kustomize]. (create and update the the `config/` directory). This plugin is used in the composition to create the plugin (`go/v3`).                      |
+| [kustomize.common.kubebuilder.io/v1](kustomize-v2-alpha.md) | `kustomize/v2-alpha` | It has the same purpose of `kustomize/v1`. However, it works with [kustomize][kustomize] version `v4` and address the required changes for the future kustomize configurations. It will probably be used with the future `go/v4-alpha` plugin. |
+| `base.go.kubebuilder.io/v3`                                 | `base/v3`            | Responsible for scaffold all files which specific requires Golang. This plugin is used in the composition to create the plugin (`go/v3`)                                                                                                       |
+| [grafana.kubebuilder.io/v1-alpha](grafana-v1-alpha.md)      | `grafana/v1-alpha`   | Optional plugin which can be used to scaffold Grafana Manifests Dashboards for the default metrics which are exported.                                                                                                                         |
```

Comment:
Nit
```suggestion
| [grafana.kubebuilder.io/v1-alpha](grafana-v1-alpha.md)      | `grafana/v1-alpha`   | Optional plugin which can be used to scaffold Grafana Manifests Dashboards for the default metrics which are exported by controller-runtime.                                                                                                                         |
```

Let's just mention default controller-runtime metrics for now. Later when we are good, we will get to a point to explain custom metrics.